tool/gravity/cli/clusterupdate.go

```diff
@@ -66,9 +66,6 @@ func updateTrigger(
 		return trace.Wrap(err)
 	}
 	defer updater.Close()
-	if !manual {
```

Comment:
Left-over from previous refactoring gone wrong - I think the second `if` makes more sense.